src/os/bluestore/bluestore_types.h

```diff
@@ -156,7 +157,7 @@ WRITE_CLASS_DENC(bluestore_pextent_t)
 
 ostream& operator<<(ostream& out, const bluestore_pextent_t& o);
 
-typedef mempool::bluestore_meta_other::vector<bluestore_pextent_t> PExtentVector;
+typedef boost::container::small_vector<bluestore_pextent_t, 1> PExtentVector;
```

Comment:
Another point here is that my PR#13337 will invalidate your assumption that pextent vector size tend to be equal to 1. Multiple pextents per single blob will be a regular case.